src/Commands/ImportDump.php

```diff
@@ -39,6 +40,9 @@ class ImportDump extends Command
      */
     protected $description = 'Imports a dump.';
```

Comment:
Since `protector` is already not giving a hint on what the package actually does, shouldn't this at least say `Imports a local or remote database dump.`?